src/Staking.sol

```diff
@@ -196,45 +213,45 @@ contract StakeRegistry is AccessControl, Pausable {
     ////////////////////////////////////////
 
     /**
-     * @dev Checks to see if `overlay` is frozen.
-     * @param overlay Overlay of staked overlay
+     * @dev Checks to see if `address` is frozen.
+     * @param _owner owner of staked address
      *
      * Returns a boolean value indicating whether the operation succeeded.
      */
-    function overlayNotFrozen(bytes32 overlay) internal view returns (bool) {
-        return stakes[overlay].lastUpdatedBlockNumber < block.number;
+    function addressNotFrozen(address _owner) internal view returns (bool) {
+        return stakes[_owner].lastUpdatedBlockNumber < block.number;
     }
 
     /**
-     * @dev Returns the current `stakeAmount` of `overlay`.
-     * @param overlay Overlay of node
+     * @dev Returns the current `stakeAmount` of `address`.
+     * @param _owner _owner of node
      */
-    function stakeOfOverlay(bytes32 overlay) public view returns (uint256) {
-        return stakes[overlay].stakeAmount;
+    function stakeOfAddress(address _owner) public view returns (uint256) {
+        return stakes[_owner].stakeAmount;
     }
 
     /**
-     * @dev Returns the current usable `stakeAmount` of `overlay`.
+     * @dev Returns the current usable `stakeAmount` of `address`.
      * Checks whether the stake is currently frozen.
-     * @param overlay Overlay of node
+     * @param _owner owner of node
      */
-    function usableStakeOfOverlay(bytes32 overlay) public view returns (uint256) {
-        return overlayNotFrozen(overlay) ? stakes[overlay].stakeAmount : 0;
+    function usableStakeOfAddress(address _owner) public view returns (uint256) {
+        return addressNotFrozen(_owner) ? stakes[_owner].stakeAmount : 0;
     }
 
     /**
-     * @dev Returns the `lastUpdatedBlockNumber` of `overlay`.
+     * @dev Returns the `lastUpdatedBlockNumber` of `address`.
      */
-    function lastUpdatedBlockNumberOfOverlay(bytes32 overlay) public view returns (uint256) {
-        return stakes[overlay].lastUpdatedBlockNumber;
+    function lastUpdatedBlockNumberOfOverlay(address _owner) public view returns (uint256) {
```

Comment:
if this is ever called on its own, it should check the overlay too. 
If not, then it implies two map lookups and should be considered spurious gas spending.

In fact I think this check should just be part of the useablestake call and be checked internally not externally

Reply:
hmm should be called ofAddress, also its read function that is useful for the debugging state of stake.
In redis could be called as part of reading of address mapping and comparing values.